src/renderer/utils/flatpak.js

```diff
@@ -0,0 +1,119 @@
+// -*- mode: js-jsx -*-
+/* Chrysalis -- Kaleidoscope Command Center
+ * Copyright (C) 2020-2022  Keyboardio, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License as published by the Free Software
+ * Foundation, version 3.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import fs from "fs";
+import path from "path";
+import readline from "readline";
+
+const flatpakInfoPath = "/.flatpak-info";
+
+async function insideFlatpak() {
+  return (
+    process.platform == "linux" &&
+    (await fs.promises.access(flatpakInfoPath).catch(() => {
```

Comment:
This doesn't seem to work for me, ended up with 
```
async function insideFlatpak() {
  return process.platform === 'linux' && fs.existsSync(flatpakInfoPath);
}
```

Reply:
I have no clue what I'm doing when it comes to Javascript, so not too surprised it didn't work. I've update this as you've suggested. Thanks!